src/main/java/tc/oc/pgm/api/Permissions.java

```diff
@@ -30,6 +30,8 @@
   String DEBUG = ROOT + ".debug"; // Errors from map loading and debug commands
   String STAFF = ROOT + ".staff"; // Considered apart of the staff team
   String RELOAD = ROOT + ".reload"; // Reload the PGM configuration
+  String FREEZE = ROOT + ".freeze"; // Can freeze players
+  String FREEZE_EXEMPT = ROOT + ".freeze.exempt"; // Cannot be frozen
```

Comment:
Don't really need this, just have the exempt be `STAFF`.

Reply:
If a player has the `FREEZE` permission, they can't be frozen.

Reply:
We might not want this, e.g. referees cannot be frozen but might not want them showing up in /staff or otherwise having the pgm.staff permissions

Reply:
So then it would make sense that if a player has the `FREEZE` permission, they can't be frozen? seems reasonable to me.

Reply:
not necessarily, we wouldn't want map developers having permission to freeze but we definitely want them to be excempt

Reply:
So what is the final decision on permissions?

Reply:
It's hard to justify having a special permission *just* to be exempt. If you can freeze someone, makes sense you can't be frozen yourself (since you could just unfreeze yourself).

So let's just exempt those with the `FREEZE` permission.

Reply:
Keep in mind `Permissions` was designed to be really simple, that's why there isn't a permission node for every nook and cranny.